Firebase/Core/Private/FIRLogger.h

```diff
@@ -79,6 +79,12 @@ void FIRSetLoggerLevel(FIRLoggerLevel loggerLevel);
  */
 BOOL FIRIsLoggableLevel(FIRLoggerLevel loggerLevel, BOOL analyticsComponent);
 
+/**
+ * Register version to include in logs.
```

Comment:
Which "version"? Is it the Firebase-wide version? The Core version?

Also, use `@param` to introduce the parameter description.